src/backend/drm/compositor/mod.rs

```diff
@@ -642,48 +642,60 @@ type RenderFrameErrorType<A, F, R> = RenderFrameError<
     R,
 >;
 
+#[derive(Debug)]
+/// Defines the element for the primary plane in cases where a new buffer was composited
```

Comment:
This is not necessarily true, it may also reference a previous buffer if there was no damage on the primary plane.